apps/vs-code-designer/src/README.md

```diff
@@ -4,6 +4,10 @@ In Visual Studio Code, you can use the Azure Logic Apps (Standard) extension to
 
 > Sign up today for your free Azure account and receive 12 months of free popular services, $200 free credit and 25+ always free services 👉 [Start Free](https://azure.microsoft.com/free/open-source).
 
+## Azure Logic Apps - Improved Onboarding Experience
+
+Starting with version 2.81.5, the Azure Logic Apps (Standard) extension for Visual Studio Code includes a dependency installer that automatically installs all the required dependencies in a new binary folder and leaves any existing dependencies unchanged. For more information, see [Get started more easily with the Azure Logic Apps (Standard) extension for Visual Studio Code](https://techcommunity.microsoft.com/t5/azure-integration-services-blog/making-it-easy-to-get-started-with-the-azure-logic-apps-standard/ba-p/3979643).
```

Comment:
@ecfan Could you please review this text? @wsilveiranz We will problably need a go link for the documentation?

Reply:
Looks good to me as this paragraph came from the official doc: https://learn.microsoft.com/azure/logic-apps/create-single-tenant-workflows-visual-studio-code

Reply:
#assign:ecfan